sdk/storage/azure-storage-queue/tests/test_queue.py

```diff
@@ -963,6 +964,17 @@ def test_unicode_update_message_unicode_data(self, resource_group, location, sto
         self.assertIsInstance(message.expires_on, datetime)
         self.assertIsInstance(message.next_visible_on, datetime)
 
+    @ResourceGroupPreparer()
+    @StorageAccountPreparer(name_prefix='pyacrstorage')
+    def test_transport_closed_only_once(self, resource_group, location, storage_account, storage_account_key):
```

Comment:
nit: you never consume location/resource_group.  Do you need them? (and in test_queue_async)